PSSwagger/Get-TaskResult.ps1

```diff
@@ -0,0 +1,104 @@
+Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
+
+<#
+.DESCRIPTION
+   Get zero or more result items from a task. Optionally, skip N items or take only the top N items. If paged, assigns $PageResult.Result
+   the page item result, and the returned result items are the items within the page.
+
+.PARAMETER  TaskResult
+    The started Task.
+
+.PARAMETER  SkipInfo
+    Object containing skip parameters or $null. Should contain the properties: 'Count', 'Max'
+
+.PARAMETER  TopInfo
+    Object containing top parameters or $null. Should contain the properties: 'Count', 'Max'
+
+.PARAMETER  PageResult
+    Object containing page result. Should contain the property: 'Result'
+
+.PARAMETER  PageType
+    Expected type of task result when the result is a page.
+#>
+function Get-TaskResult {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [object]
+        $TaskResult,
+
+        [Parameter(Mandatory=$false)]
+        [PSObject]
+        $SkipInfo = $null,
```

Comment:
It looks like this parameter type should be PSCustomObject, right?

Minor: Also $null assignment can be removed as optional param value is null by default. 

Consider adding checks on existence of Count and Max members/keys of $SkipInfo and $TopInfo?